metadata.json

```
@@ -0,0 +1,29 @@
+{
+  "name": "puppetlabs-powershell_task_helper",
+  "version": "0.1.0",
+  "author": "puppetlabs",
+  "summary": "",
+  "license": "Apache-2.0",
+  "source": "",
```

```suggestion
  "source": "https://github.com/puppetlabs/puppetlabs-powershell_task_helper",
```

Our release job fails without this.